### PR TITLE
Fixed failing functional tests for compose.

### PR DESCRIPTION
**Pull request by dipak-pawar**
#### Short description of what this resolves:
Fixed failing tests for docker-compose.

#### Changes proposed in this pull request:

- tomcat docker image is using jre8

**Fixes**: #548

### CODE REVIEW

File: docker/ftest-docker-compose-v2/docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '2'
 services:
   tomcat:
     env_file: envs
-    image: tutum/tomcat:7.0
```
Comment (lordofthejars) [inline docker/ftest-docker-compose-v2/docker-compose.yml]: why not just use tomcat:jre8 directly here?

Comment (dipak-pawar) [inline docker/ftest-docker-compose-v2/docker-compose.yml]: We have to add users explicitly by adding `ADD tomcat-users.xml conf/` in dockerfile, which is not provided in base image. ref - https://hub.docker.com/_/tomcat/

Comment (lordofthejars) [inline docker/ftest-docker-compose-v2/docker-compose.yml]: ah right, now I understand the problem.

**Comment by lordofthejars** [comment]
Good work, thank you so much.

**Pull request merged**